src/EditorFeatures/Core.Wpf/NavigateTo/NavigateToItemDisplay.cs

```diff
@@ -105,7 +105,8 @@ public void NavigateTo()
             // Document tabs opened by NavigateTo are carefully created as preview or regular
             // tabs by them; trying to specifically open them in a particular kind of tab here
             // has no effect.
-            navigationService.TryNavigateToSpan(workspace, document.Id, _searchResult.NavigableItem.SourceSpan);
+            // TODO: Get the platform to use and pass us an operation context, or create one ourselves.
+            navigationService.TryNavigateToSpan(workspace, document.Id, _searchResult.NavigableItem.SourceSpan, CancellationToken.None);
```

Comment:
This is a negative case.  We did non-cancellable work before, and we still do that now.  But at least now this is pushed up to the edge.  Ideally, we can work with the platform to make this a TWD location.